src/styles/components/_toc.scss

```diff
@@ -0,0 +1,162 @@
+@import '../../styles/settings/colors';
+@import '../../styles/settings/type';
+@import '../../styles/tools/breakpoints';
+@import '../../styles/tools/mixins';
+
+@mixin TOC_LINKS {
+  @include bp(md) {
+    color: $PRIMARY_TEXT_COLOR;
+
+    &.w-toc__active--link,
+    &:hover {
+      color: $WEB_PRIMARY_COLOR;
+    }
+  }
+}
+
+@mixin TOC_PADDING {
+  padding: 0 24px;
+}
+
+.w-toc__active--border {
+  border-left: 3px solid $WEB_PRIMARY_COLOR;
+  margin-left: -24px;
+  padding-left: 21px;
+}
+
+div#content.w-toc-open {
+  > * {
+    @include bp(md) {
+      width: calc(100% - 268px);
```

Comment:
I think you may want to put the ToC width into a variable. Because it's so important you might want to put it in settings/_dimensions.scss. That's where the variable for the topnav is.

Reply:
I also feel a bit weird about this rule. It clearly works, but it's also very broad.
I was wondering if this could be done with flexbox by giving the toc a flex-basis to 268px when it's open and setting its flex-shrink to 0. Might need to fiddle around with it a bit, but this might be safer. For instance, we might put an absolutely positioned child in `#content` and we wouldn't want its width to get broken when the ToC is open.